.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+References CAR-XXXX
+
+## Motivation and context
+
+[Describe the problem we're solving and lay out the solution]
+
+## Before
+
+[Describe the current behavior and / or add a screenshot of the current state]
+
+## After
+
+[Describe the new behavior and / or add a screenshot of the new state]
```

Comment:
before/after makes a lot of sense for visual changes. for such technical project, I'd consider skipping it